common/h/registers/x86_64_regs.h

```diff
@@ -266,6 +274,8 @@ namespace Dyninst { namespace x86_64 {
   DEF_REGISTER(       st5,    0x5 | FPDBL |   X87 | Arch_x86_64, "x86_64");
   DEF_REGISTER(       st6,    0x6 | FPDBL |   X87 | Arch_x86_64, "x86_64");
   DEF_REGISTER(       st7,    0x7 | FPDBL |   X87 | Arch_x86_64, "x86_64");
+  DEF_REGISTER(       fcw,    0x8 | W_REG |   X87 | Arch_x86_64, "x86_64");
+  DEF_REGISTER(       fsw,    0x9 | W_REG |   X87 | Arch_x86_64, "x86_64");
```

Comment:
There needs to be a new category type for `fcw` and `fsw` (and `mxcsr`) such as `FPCTL`, otherwise `getBaseRegister` will convert these to a register with category `<id> | W_REG | ST | Arch_x86_64` which is incorrect.  `getBaseRegister` should return the register unmodified.

Reply:
Ah, that's right. I forgot about that. I actually started out making a new category for them, but then changed my mind at the last second...